src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -469,20 +458,14 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
     void ConptyConnection::_ClientTerminated() noexcept
     try
     {
-        if (_isStateAtOrBeyond(ConnectionState::Closing))
-        {
-            // This termination was expected.
-            return;
-        }
-
         // EXIT POINT
         DWORD exitCode{ 0 };
         GetExitCodeProcess(_piClient.hProcess, &exitCode);
 
         // Signal the closing or failure of the process.
         // Load bearing. Terminating the pseudoconsole will make the output thread exit unexpectedly,
         // so we need to signal entry into the correct closing state before we do that.
-        _transitionToState(exitCode == 0 ? ConnectionState::Closed : ConnectionState::Failed);
+        _transitionToState(exitCode == 0 || exitCode == STILL_ACTIVE ? ConnectionState::Closed : ConnectionState::Failed);
```

Comment:
`STILL_ACTIVE` can occur if the CLI app used `FreeConsole()` to close the tab, while it keeps running in the background. This filters the error away.